core/templates/components/forms/forms-templates/mark-all-audio-and-translations-as-needing-update-modal.directive.html

```diff
@@ -6,11 +6,12 @@ <h3>Update Audio Translations</h3>
   <p>
     Would you like to flag all existing translations and voice-over audio as 'may not match content'?
     <br>
+    <br>
     This informs the voice artists to re-translate and record new audio that reflects any changes in text.
   </p>
 </div>
 
 <div class="modal-footer">
-  <button class="btn btn-secondary" ng-click="cancel()">No</button>
+  <button class="btn btn-secondary" ng-click="cancel()">Cancel</button>
```

Comment:
Let's keep `Cancel`. Ditto below.

Reply:
Correction: I meant, let's keep `No`. Please undo this change.

Reply:
done